source/app/service-providers/appearance-provider.ts

```diff
@@ -121,9 +121,9 @@ export default class AppearanceProvider extends EventEmitter {
               // Calculate the contrast before returning
               const dark = '33333333'
               const light = 'eeeeeeee'
-              const r = parseInt(accentColor.substring(0, 2), 16); // hexToR
-              const g = parseInt(accentColor.substring(2, 4), 16); // hexToG
-              const b = parseInt(accentColor.substring(4, 6), 16); // hexToB
+              const r = parseInt(accentColor.substring(0, 2), 16) // hexToR
+              const g = parseInt(accentColor.substring(2, 4), 16) // hexToG
+              const b = parseInt(accentColor.substring(4, 6), 16) // hexToB
```

Comment:
Agree the code does not pass lint. However our team should not fix problems in the appearance-provider because appearance-provider is not related to tray feature Zettlr#175.  I suggest don't change `appearance-provider.ts`.

Reply:
will roll back